nullaway/src/test/resources/com/uber/nullaway/testdata/NullAwayAnonymousClass.java

```diff
@@ -35,6 +35,10 @@ static void anonymousClass(final Inner x) {
 
     Runnable r =
         new Runnable() {
+
+          // BUG: Diagnostic contains: @NonNull field NullAwayAnonymousClass$1.f not initialized
+          private Object f;
```

Comment:
@msridhar I am fine with this test case here, but do you think it makes sense to piggy-back on this class or just create a new anon Runnable in `CheckFieldInitPositiveCases`?

Reply:
I think it's ok to just piggy-back the test here.  It's about anonymous classes and also field initialization, so it seems either spot would be fine.

Reply:
Fine by me. Let's just fix the other nit and land this, then!